reference/sectionC_interpretation.inc

```diff
@@ -238,7 +238,7 @@ yocto    −24
 Component reference
 -------------------
 
-A "component reference" refers to the :code:`component_ref` attribute used in :code:`import component` items, to the :code:`component_ref` elements which occurs as children of :code:`encapsulation` items, and to the :code:`component_1` and :code:`component_2` attributes used in :code:`connection` items.
+The term "component reference" refers to the value of a :code:`component_ref` attribute used in an :code:`import` item for :code:`component` items, to :code:`component_ref` elements which occur as children of :code:`encapsulation` items, and to the value of :code:`component_1` and :code:`component_2` attributes used in :code:`connection` items.
```

Comment:
Why is the wording different for units and component? (E.g., "the value of **the**" vs. "the value of **a**")

Reply:
I think it should be indefinite, so will go with **a**.

Reply:
Done in 1f6e180